lib/dishes.rb

```diff
@@ -0,0 +1,19 @@
+class Dishes
+
+  attr_accessor :dish_list
+
+  def initialize
```

Comment:
You've hardcoded your hash of dishes into the Dish class. Is there a better way of making that data available?